docs/Users/Glob_Patterns.rst

```diff
@@ -37,6 +37,11 @@ The special characters used in shell-style wildcards are:
 | ``**``            | Matches everything.                                     |
 +-------------------+---------------------------------------------------------+
 
+.. note::
+
+    If you're looking for a negation pattern to ignore files, check out the
```

Comment:
to ignore files -> to exclude files

people may look for `exclude` specifically so that keyword might trigger it